### PR TITLE
Working in OTP 24

### PR DESCRIPTION
**Pull request by matreyes**
Updated Lager version to work on OTP 24

### CODE REVIEW

File: rebar.config.script [hunk 1]
```diff
@@ -1,9 +1,9 @@
 IsRebar3 = erlang:function_exported(rebar3, main, 1),
 
 Rebar2Deps = [
-    {lager, ".*", {git, "https://github.com/erlang-lager/lager.git", {tag, "3.8.0"}}},
-    {jsone, ".*", {git, "https://github.com/sile/jsone.git", {tag, "1.5.2"}}},
-    {esq, ".*", {git, "https://github.com/fogfish/esq.git", {tag, "2.0.4"}}}
+    {lager, "3.9.2"},
```
Comment (silviucpp) [inline rebar.config.script:4]: rebar2 supports this format for dependencies ? ({lager, "3.9.2"},)

**Review by silviucpp — CHANGES_REQUESTED** [review]
Please add also OTP24 in travis config !

File: rebar.config.script (hunk 1 above)
Comment (matreyes) [reply rebar.config.script:4]: Hi! I just tried to change sources to hex, and it works, but Im not sure if it's the recommended way (I'm Elixir developer), please tell me if I'm doing it wrong.

Actually I have another problem and maybe we can fix it here:
Using Hex.pm it doesn't pull rebar.config.script, and then when running in elixir it prints a warning.

**Comment by matreyes** [comment]
As rebar3 has removed support from OTP < 22, What do you think of removing OTP 21 and before from travis?

**Comment by matreyes** [comment]
@silviucpp remember to push to hex!

And thanks for this amazing library!

**Pull request merged**